content/en/docs/concepts/storage/volumes.md

```diff
@@ -1355,37 +1358,9 @@ as usual, without any CSI specific changes.
 
 {{< feature-state for_k8s_version="v1.16" state="beta" >}}
 
-This feature allows CSI volumes to be directly embedded in the Pod specification instead of a PersistentVolume. Volumes specified in this way are ephemeral and do not persist across Pod restarts.
+This feature allows CSI volumes to be directly embedded in the Pod specification instead of a PersistentVolume. Volumes specified in this way are ephemeral and do not persist across Pod restarts. See [the ephemeral volume page](/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volume) for more information.
```

Comment:
```suggestion
You can directly configure CSI volume claims within the Pod specification. Volumes specified in this way are ephemeral and do not persist across Pod restarts. See [Ephemeral Volumes](/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volume) for more information.
```

I'm saying _volume claim_ because “ephemeral PersistentVolumeClaim” doesn't really make sense.

Reply:
In this case "CSI volumes" makes more sense than "CSI volume claims" because there is nothing resembling a "claim" in the API. The rest makes sense, changed.